Example/NYTPhotoViewer-Swift/ViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ViewController.swift
+//  NYTPhotoViewer-Swift
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, NYTPhotosViewControllerDelegate {
+
+    @IBOutlet weak var imageButton : UIButton?
+    private let photos = PhotosProvider().photos
+    private var photosViewController: NYTPhotosViewController
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
+        photosViewController = NYTPhotosViewController(photos: photos as [AnyObject]!)
```

Comment:
It seems strange to me that a constant of type `[ExamplePhoto]` would need to be cast to `[AnyObject]`. Is there any way to prevent the need for casting here and later in the file?